compiler/test/run.d

```diff
@@ -197,7 +197,7 @@ Options:
                             : "`unit` tests";
 
                 writeln(">>> TARGET FAILED: ", name);
-                failedTargets ~= name;
+                synchronized failedTargets ~= name;
```

Comment:
Just realized it would be better to declare `failedTargets` as `shared`. Can do that tomorrow.